shell/src/main/java/alluxio/shell/command/LsCommand.java

```diff
@@ -44,15 +44,15 @@
   /**
    * Formats the ls result string.
    *
-   * @param acl whether security is enabled
-   * @param isFolder whether this path is a file or a folder
-   * @param permission permission string
-   * @param userName user name
-   * @param groupName group name
-   * @param size size of the file in bytes
+   * @param acl          whether security is enabled
```

Comment:
please keep the original style, no extra spaces are needed to be aligned, apply to the whole PR.